src/rp/util/logic.clj

```diff
@@ -0,0 +1,7 @@
+(ns rp.util.logic)
+
+(defn xor
```

Comment:
i know it's minor, but since we intend to use `rp.util` across apps/sevices, I think we should aim for docstrings in all public fns.

even if it's just "Logical exclusive OR"